ParkingLot/ParkingBoy.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+namespace ParkingLotCLI
+{
+    public class ParkingBoy
+    {
+        public ParkingBoy(List<ParkingLot> parkingLots)
+        {
+            ParkingLots = parkingLots;
+        }
+
```

Comment:
Can this code create a parking boy without parking lot?

Reply:
I tested it, it will create a parking boy. @bukeyuanguan